### PR TITLE
Fix redundant module updates

### PR DESCRIPTION
**Pull request by juliangruber**
By storing the latest installed module version in a text file, prevent redundant updates.

### CODE REVIEW

File: lib/modules.js [hunk 1]
```diff
@@ -115,19 +115,34 @@ async function getLatestDistTag (repo) {
   return body.tag_name
 }
 
-const lastModuleDistTag = new Map()
+async function getLastSeenModuleDistTag ({ module, latestModuleVersions }) {
```
Comment (bajtos) [inline lib/modules.js:118]: I find it a bit confusing that `latestModuleVersions` is a directory path. When I see this name, I expect a collection of version numbers.

How about changing this name to something like `latestVersionsDir` or `moduleVersionsDir`? `latestModuleVersionsDir` is an option too, although a bit long.

**Review by bajtos — APPROVED** [review]
Nice!

File: lib/modules.js (hunk 1 above)
Comment (juliangruber) [reply lib/modules.js:118]: I found this confusing as well, thanks will change!

**Review by bajtos — APPROVED** [review]
:shipit:

**Pull request merged**